ios/RNBranch.m

```diff
@@ -28,6 +29,22 @@
 static NSString * const RNBranchErrorDomain = @"RNBranchErrorDomain";
 static NSInteger const RNBranchUniversalObjectNotFoundError = 1;
 
+static NSString * const MINIMUM_REQUIRED_BRANCH_SDK = @"0.26.0";
+
+static BOOL branchSdkVersionValid() {
+    NSArray<NSString *> *components = [BNC_SDK_VERSION componentsSeparatedByString:@"."];
+    int major = components[0].intValue;
+    int minor = components[1].intValue;
+    int patch = components[2].intValue;
+
+    NSArray<NSString *> *required = [MINIMUM_REQUIRED_BRANCH_SDK componentsSeparatedByString:@"."];
+    int requiredMajor = required[0].intValue;
+    int requiredMinor = required[1].intValue;
+    int requiredPatch = required[2].intValue;
+
+    return major > requiredMajor || (major == requiredMajor && (minor > requiredMinor || (minor == requiredMinor && patch >= requiredPatch)));
```

Comment:
I'm not sure if there is a more standard way in Obj-C to compare version numbers. This was straightforward but off the cuff.